packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -142,28 +106,11 @@ registerStyles(
       opacity: 0;
     }
 
-    /* Read-only */
-    :host([readonly]) [part='input-field'] {
-      color: var(--lumo-secondary-text-color);
-      background-color: transparent;
-      cursor: default;
-    }
-
-    :host([readonly]) [part='input-field']::after {
-      background-color: transparent;
-      opacity: 1;
-      border: 1px dashed var(--lumo-contrast-30pct);
-    }
```

Comment:
This CSS is moved here:
https://github.com/vaadin/web-components/blob/91bfb8acc231ec7747bbebbfe2add5b83e647e00/packages/input-container/theme/lumo/vaadin-input-container-styles.js#L49-L59